### DISCUSSION TITLE
Calibration Window

### DISCUSSION THREAD

**Discussion by TarcisoSilva** [NightscoutFoundation/xDrip#2039] [Ideas]
Hi.
Thanks for XDRIP it is a great tool to help me to controle my BG.

I would like to recommend a new feature to XDRIP. I would like to see one new type of alert. 

Requiremets:
This new alarm will be trigger on business hours
This alarm will be trigger after 30 minutes os flat BG
It should be optional on Extra Alerts  

Thanks

**Comment by Navid200** [comment]
30 minutes of flat BG?
What would it be useful for?

**Reply by TarcisoSilva** [reply]
30 minutes on flat BG should give some good window to calibration.
Some users are not aware about the best moment to calibration. On this case it is better to have this alert like a remminder. "This is a good time to calibration". Normally some user try to calibration on wrong momments with BG going up or down. So it could help to teach users that they need to calibration on the FLAT BG.

**Reply by Navid200** [reply]
Makes sense.

**Reply by ANDREAPIPPI** [reply]
Hi. I agree with TarcisoSilva, because a long time ago I asked for the same thing, i.e. an alert that asks my attention to the possibility to make a good calibration. I was asking this could happen after at least 15 minutes with a max delta of 10mg between start/end of the checked time, so I join this request. Thank you.   Andrea

**Comment by Navid200** [comment]
I have reopened the feature request:
https://github.com/NightscoutFoundation/xDrip/issues/1006
